package.json

```diff
@@ -17,6 +17,7 @@
     "express-session": "^1.15.6",
     "method-override": "^3.0.0",
     "mongoose": "^5.4.12",
+    "nodemon": "^1.18.10",
```

Comment:
nodemon is now both a dependency and a dev dependency. Since we only need nodemon for our local machine we can delete it from the dependencies (but not dev dependencies).